pallets/asset/src/lib.rs

```diff
@@ -175,7 +171,6 @@ pub mod pallet {
 
 	/// asset vc entry identifiers with details stored on chain.
 	#[pallet::storage]
-	#[pallet::getter(fn vc_issued)]
```

Comment:
Some of the places we were using custom getters. need to check the SDK impact of this change as well